scripts/restore.sh

```diff
@@ -175,25 +152,18 @@ fi
 
 echo docker cp ${CONTAINER_NAME}:/ot-node/current/config/config.json ./
 docker cp ${CONTAINER_NAME}:/ot-node/current/config/config.json ./
-# cp ~/ot-node/config/config.json ./
-
-databaseName=($(cat config.json | grep "\"database\": \"" | sed -r "s_([[:blank:]]+)\"database\":[[:blank:]]\"__" | sed "s/\",$//"))
-databaseName=${databaseName[@]:${environmentIndex}:1}
-echo "database name ${databaseName}"
-
-databaseUsername=($(cat config.json | grep "\"username\": \"" | sed -r "s_([[:blank:]]+)\"username\":[[:blank:]]\"__" | sed "s/\",$//"))
-databaseUsername=${databaseUsername[@]:${environmentIndex}:1}
-echo "database username ${databaseUsername}"
-
-databasePassword=($(cat config.json | grep "\"password\": \"" | sed -r "s_([[:blank:]]+)\"password\":[[:blank:]]\"__" | sed "s/\",$//"))
-databasePassword=${databasePassword[@]:${environmentIndex}:1}
-echo "database password read from configuration"
 
 rm config.json
 
 echo "cp -r ${BACKUPDIR}/arangodb ${temp_folder}/"
 cp -r ${BACKUPDIR}/arangodb ${temp_folder}/
 
+databaseName=$(cat ${BACKUPDIR}/arangodb/database.txt)
+echo "database name ${databaseName}"
+
+databaseUsername=$(cat ${BACKUPDIR}/arangodb/username.txt)
+echo "database username ${databaseUsername}"
```

Comment:
This is working on the assumption that the database name and username will not be changed between the backup and the restore, correct? If we're okay with this approach consider it approved

Reply:
Yes that's the assumption